pkg/distro/rhel/rhel9/gce.go

```diff
@@ -25,6 +24,7 @@ func mkGCEImageType() *rhel.ImageType {
 		[]string{"archive"},
 	)
 
+	it.NameAliases = []string{"gce-rhui"}
```

Comment:
Just a simple question here for my own sake. Does this make the changes backwards compatible? I.e. is the idea that we could still request the `gce-rhui` type and we would get the `gce` image type instead?

Reply:
Backward compatible in the sense that the `gce` image type will be used if `gce-rhui` is requested. But the image won't have any RHUI bits installed by default. 

So "kind of" to your first question and "yes" to the second.

Reply:
Cool cool, yeah that was my understanding. Thanks for clarifying